corehq/apps/es/case_search.py

```diff
@@ -283,10 +283,7 @@ def _case_property_not_set(case_property_name):
     return filters.NOT(
         queries.nested(
             CASE_PROPERTIES_PATH,
-            queries.filtered(
-                queries.match_all(),
-                filters.term('{}.key.exact'.format(CASE_PROPERTIES_PATH), case_property_name),
-            )
+            filters.term(PROPERTY_KEY, case_property_name),
```

Comment:
fyi I resolved a conflict with master by mixing @esoergel's recent changes with the change you have in this PR @Charl1996. Just wanted to draw attention to this in case it needs to change.